SampleApplications/Samples/NetCoreConsolePublisher/Program.cs

```diff
@@ -225,7 +225,7 @@ private static void CleanupPublisher()
                 false,
                 m_configuration.ApplicationName,
                 60000,
-                null,
+                new UserIdentity(new AnonymousIdentityToken()),
```

Comment:
Another option might be to do a if (!= null) in Session.Create and create the identity.